### PR TITLE
Cover alerta page view

### PR DESCRIPTION
**Pull request by israelst**
This PR is a by product of the process of creating a new view of the data. More PRs coming soon :).

I had to understand and change the Django View that feed the map present at site home page to support my new visualization, so write tests to it. As it has its own importance, I'm submitting this changes before I finish the whole feature.

### CODE REVIEW

File: AlertaDengue/dados/views.py [hunk 1]
```diff
@@ -34,19 +34,12 @@ def get_context_data(self, **kwargs):
...
-        #                   sum(casos_ap.values()),
-        #                   semana, data))
         total_series = np.zeros(len(case_series['4.0']))
         for s in case_series.values():
             total_series += np.array(s)
-        print(str(total_series.tolist())[1:-1])
         context.update({
             'casos_por_ap': json.dumps(casos_ap),
             'alerta': alerta,
-            'semana': semana,
```
Comment (flavioamieiro) [inline AlertaDengue/dados/views.py]: I don't know the code base enough, so I don't know why this attribute should be removed. Just commenting to check.

Comment (israelst) [inline AlertaDengue/dados/views.py]: >  EDIT: I'm assuming it's because 'SE' is already included and is the one used.

Yes!

**Comment by flavioamieiro** [comment]
I love this pull request. It's an important clean up and these tests, if ran with each new dataset, should catch errors like the one we had a few days back where the dataset didn't include one of the fields. In my opinion we should merge it right away :)

File: AlertaDengue/dados/tests.py [hunk 2]
```diff
@@ -1,4 +1,85 @@
 # coding=utf-8
+import json
+
+from django.core.urlresolvers import reverse
 from django.test import TestCase
 
-# Create your tests here.
+class TestAlertaPageView(TestCase):
+    def setUp(self):
+        # TODO: Create fixtures to prevent test fails in case of data changing
```
Comment (fccoelho) [inline AlertaDengue/dados/tests.py]: I think we finish the fixtures before merging so that we are independent of current test dataset data.

**Comment by fccoelho** [comment]
Changes seem safe, so when we have the fixtures, we can merge.

**Comment by israelst** [comment]
What do you guys think, can we merge?

**Comment by flavioamieiro** [comment]
The fixtures are there so I think we can merge, yes.

**Comment by fccoelho** [comment]
Go ahead and merge.
Em 11/06/2015 00:01, "Flávio Amieiro" <notifications@github.com> escreveu:

**Comment by israelst** [comment]
I'm not very comfortable with merging my own PR but, as you all have agreed, I'll do that.

**Pull request merged**